trio_async/workflows.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
+
+from temporalio import workflow
+
+with workflow.unsafe.imports_passed_through():
+    from trio_async.activities import say_hello_activity_async, say_hello_activity_sync
+
+
+@workflow.defn
+class SayHelloWorkflow:
+    @workflow.run
+    async def run(self, name: str) -> list[str]:
+        # Workflows don't use default asyncio event loop or Trio, they use a
+        # custom event loop. Therefore Trio primitives should never be used in a
+        # workflow, only asyncio helpers (which delegate to the custom loop).
```

Comment:
These comments are very helpful. Two teaks: if "primitives" and "helpers" are the same thing can we use the same word, and can you give an example so that it's really clear what we can and can't do. Something like this:

```suggestion
        # custom event loop. Therefore Trio primitives such as XXX should never be used in a
        # workflow, only functions and classes from the asyncio module, (which delegate to the custom loop).
```